addons/account/wizard/accrued_orders.py

```diff
@@ -157,7 +157,7 @@ def _ellipsis(string, size):
         for order in orders:
             if len(orders) == 1 and self.amount and order.order_line:
```

Comment:
if there is only one section order line (not sure it's really possible through the UI ; but anyway, let's make things more robust :) ), you'll still enter this condition, then your [0] below will crash. You should retrieve the order lines with your filtered() before this condition, and only enter it if this filtered() gave a result.